server/utils/utils.ts

```diff
@@ -137,6 +137,6 @@ export const calculateAge = (dateOfBirth: string) => {
   return moment().diff(dateOfBirth, 'years')
 }
 
-export const generateCurrentYear = () => {
-  return moment().year()
+export const generateCurrentYear = (now: moment.Moment = moment()) => {
```

Comment:
does this moment get created at start up or on every method call? 
You'd want it to be every method call, so maybe a function would be better `nowGetter = () => moment`

where the impl of generateCurrentYear would be: 
nowGetter().year()

Reply:
Think this is now sorted in latest commit